rend3-gltf/src/lib.rs

```diff
@@ -102,6 +118,10 @@ pub struct Texture {
     pub format: types::TextureFormat,
 }
 
+#[derive(Debug)]
+pub struct Skin {
+    pub inverse_bind_matrices: Vec<Mat4>,
+}
```

Comment:
The skin also has joint information we probably want to expose later. For now I still haven't figured out how to interpret that data, so we can decide how to best expose it in another PR